baystation12.dme

```diff
@@ -5,6 +5,87 @@
 // END_INTERNALS
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"

```

Comment:
You have to disable automatically adding FILE_DIR.
Build > preferences for baystation12.dme > untick the box.